README.md

```diff
@@ -144,15 +160,19 @@ highlighting and NLS.
 
 #### Formatting
 
-You can format Nickel source code using [Topiary](https://github.com/tweag/topiary/):
+To format a Nickel source file, use `nickel format` (`-i` is short for
+`--in-place`):
 
 ```console
-topiary -i -f my-config.ncl
+nickel format -i my-config.ncl
```

Comment:
```suggestion
nickel format -i -f my-config.ncl
```
Ideally, I'd like to change that to `nickel format my-config.ncl` with support for multiple files in the same call as well, but for now the CLI works like this.